apps/base-docs/docs/pages/identity/smart-wallet/features/app-accounts.mdx

```diff
@@ -0,0 +1,9 @@
+# App Accounts
+
+App Accounts enable developers to create and manage app-specific accounts for their applications. These accounts are linked to the user's Smart Wallet and can be utilized to sign transactions specifically for the app. This integration grants developers full control over the user experience, ensuring a seamless and cohesive interaction within the app.
+
+## Technical Details
+
+- Smart Wallet Ownership: The user's Smart Wallet acts as an owner of the app account, allowing it to manage and control the app-specific account.
+- Key Agnostic: App Accounts are designed to be key agnostic. They leverage our Linked Accounts technology, which supports various key types including in-browser CryptoKeys, Passkeys, and server signer products.
```

Comment:
we need a page on different signers types, and the reasons why you as a developer might want to use different ones based your app's use case

Reply:
Good callout. I agree that this could be beneficial to developers